docs/app/Components/TAAttribution/TAAttribution.js

```diff
@@ -68,7 +68,7 @@ class TAAttribution extends Component {
           inline
           verticalAlign='middle'
           style={imageStyle}
-          src={`${__BASE__}assets/technologyadvice-logo-dark.png`}
+          src={'/assets/technologyadvice-logo-dark.png'}
```

Comment:
`src={'/assets/technologyadvice-logo-dark.png'}` can be replaced with `src='/assets/technologyadvice-logo-dark.png'`